<launch>
<node name="autocirc" pkg="autocircle_generater" type="talkerAutoCircle" output="screen"/>
</launch>
